### PR TITLE
Attu change

### PR DESCRIPTION
**Pull request by nameczz**
Support some new features for Milvus 2.1.0

### CODE REVIEW

File: site/en/userGuide/tools/attu.md [hunk 1]
```diff
@@ -18,7 +18,10 @@ Attu is under rapid development and new features are added on a weekly basis. We
 
 Below is the features we have to offer:
 
-- View milvus cluster statistics in a glance. (TBD)
+- Support username, password, and TLS connection
+  ![Attu_Login](../../../../assets/attu/insight_install.png "Connect to the Attu service.")
+  ![Attu_Login_user_pwd](../../../../assets/attu/insight_install_user_pwd.png "Connect to the Attu service with user")
```
Comment (YiyunNi) [inline site/en/userGuide/tools/attu.md:23]: Attu_Login_user_pwd

File: site/en/userGuide/tools/attu_install-docker.md [hunk 2]
```diff
@@ -17,20 +17,24 @@ This topic describes how to install Attu, an efficient open-source management to
 
 - Milvus installed on [your local device](install_standalone-docker.md) or [cluster](install_cluster-docker.md).
 - Docker 19.03 or later
+- Attu v2.1.0 or later
+- Milvus 2.1.0 or later
 
 <div class="alert note">
-Attu only supports Milvus 2.x.
+If useing Milvus 2.0.x , you can check <a href="https://milvus.io/docs/v2.0.x/attu_install-docker.md">v2.0.x attu doc</a>.
```
Comment (YiyunNi) [inline site/en/userGuide/tools/attu_install-docker.md:24]: See <a href="https://milvus.io/docs/v2.0.x/attu_install-docker.md">v2.0.x Attu doc</a> if you are using Milvus 2.0.x.

File: site/en/userGuide/tools/attu_system.md [hunk 3]
```diff
@@ -16,16 +16,8 @@ Click the **System View** icon on the left-side navigation pane to visit the Sys
...
-- **Memory**: Shows the usage of memory.
-- **Qps**: Shows the last 10 monitored QPS. Each node represents a time point. Hover on a node to view the QPS monitored at the time point.
-- **Latency**: Shows the last 10 monitored latency. Each node represents a time point. Hover on a node to view the latency monitored at the time point.
-
-![System View](../../../../assets/attu/insight_system2.png "Latency at 25/11/2021, 10:51:52.")
-![System View](../../../../assets/attu/insight_system3.png "Latency at 25/11/2021, 10:55:52.")
-
 - **Topology**: Shows the structure of your Milvus instance. Click Milvus node or a coordinator node and the corresponding information appears in the Info tile on the right.
-- **Info**: Shows the hardware, system, and configuration information of the selected node.
+- **Info**: Shows the system, and configuration information of the selected node.
```
Comment (YiyunNi) [inline site/en/userGuide/tools/attu_system.md:20]: **Info**: Shows the system and configuration information of the selected node.  (Please remove the comma in sentence).

**Comment by nameczz** [comment]
@YiyunNi  please check again

**Pull request merged**